src/dbnode/storage/namespace.go

```diff
@@ -116,6 +116,9 @@ type dbNamespace struct {
 	// entry will be nil when this shard does not belong to current database
 	shards []databaseShard
 
+	// contains the schema registry for the database namespace.
```

Comment:
same nit, sorry the existing comments are really old

Reply:
Also this comment could be a little more helpful: "Contains the schema registry for the database namespace. Can be updated dynamically and will propagate to existing streams lazily on subsequent writes." Or whatever information you think might be useful here

Reply:
@Haijuncao bump

Reply:
ok